crds/virtualmachines.yaml

```diff
@@ -1156,8 +1158,7 @@ spec:
                   description: Resource utilization statistics.
                   properties:
                     cpu:
-                      description: CPU utilization statistics.
-                        usage.
+                      description: CPU utilization statistics. usage.
```

Comment:
Weird description =)

Reply:
I've not touching this, only formatter do his dirty work)

Reply:
@yaroslavborbat something strange here, what happens with this description?

Reply:
This description is not generated as I thought. It's ok.